### PR TITLE
aligned data resiliency,capacity and performance cards

### PR DESCRIPTION
**Pull request by afreen23**
alignednow

### CODE REVIEW

File: sass/components/Dashboard/utilization.scss [hunk 1]
```diff
@@ -35,3 +35,7 @@
   border-left-style: solid;
   border-left-color: rgb(209, 209, 209);
 }
+
+.kubevirt-utilization__card {
```
Comment (suomiy) [inline sass/components/Dashboard/utilization.scss]: I don't think this is gonna work on all devices. + this does not belong to `Dashboard/`

**Comment by suomiy** [comment]
why does the performance need to be the same height in the first place?  Also, if you want the resiliency and capacity the same height, you should put them in the same card and fix them with css (they can still look like two cards).

**Comment by suomiy** [comment]
>  , you should put them in the same card grid

please see Health & Compliance in ClusterOverview for inspiration

**Comment by andybraren** [comment]
> > , you should put them in the same card grid
> 
> please see Health & Compliance in ClusterOverview for inspiration

Looping in @yuvalgalanti. I wasn't a part of these discussions, but I believe the group decided to visually separate the Health and Data Resiliency cards to not imply any connection between the two. The combined Health/Compliance card of the Cluster Dashboard can stay as-is for now.

**Comment by jelkosz** [comment]
Looping @matthewcarleton in since he is now enhancing the styles to make sure it is in line with his work.

**Comment by suomiy** [comment]
> the group decided to visually separate the Health and Data Resiliency cards to not imply any connection between the two. 

I wasn't suggesting that.  I just mentioned Health & Compliance as an example how to do the layout.

**Comment by afreen23** [comment]
should I close this PR since for now alignment is not possible ..in feedback somebody suggested but on setting that recovery rate is cutting...

**Comment by andybraren** [comment]
@suomiy Ah, I misunderstood, thank you for clarifying.

**Comment by matthewcarleton** [comment]
@afreen23 I think we can close this now. I've updated the layout to just use a grid gap which helps with the spacing.

**Pull request closed without merging**